code/script.js

```diff
@@ -1,9 +1,19 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+const accordionQuestions = document.querySelectorAll('.question');
+ 
+accordionQuestions.forEach(question => {
+  question.onclick = event => {
+    const isActive = question.classList.contains('active')
+    accordionQuestions.forEach(inner => {
+      inner.classList.remove('active')
+    })
+    if(isActive===false){
+      question.classList.toggle('active')
+    }
+    
+  }
+
+})
```

Comment:
Well done finding a solution to your problem by yourself, keep up the good work! 🏆